language-testutils/src/test/java/de/jplag/testutils/LanguageModuleTest.java

```diff
@@ -95,7 +96,7 @@ final void testSourceCoverage(TestData data) throws ParsingException, IOExceptio
         this.configureIgnoredLines(ignoredLines);
         List<Integer> relevantLines = new ArrayList<>(ignoredLines.getRelevantLines());
 
-        tokens.stream().map(Token::getLine).forEach(relevantLines::remove);
+        tokens.stream().flatMap(t -> IntStream.rangeClosed(t.getStartLine(), t.getEndLine()).boxed()).forEach(relevantLines::remove);
```

Comment:
Is `rangeClosed` what we want here? I would think that `List.of(t.getStartLine(), t.getEndLine())` is more helpful, but I might well be wrong!